boards/storm/Makefile

```diff
@@ -22,30 +26,51 @@ doc:
 .PHONY: target/sam4l/release/storm
 target/sam4l/release/storm:
 	@cargo build --release --target=sam4l.json
-	@$(OBJDUMP) $(OBJDUMP_FLAGS) $@ > target/sam4l/release/storm.lst
+	@$(SIZE) $@
+
+.PHONY: target/sam4l/debug/storm
+target/sam4l/debug/storm:
+	@cargo build --target=sam4l.json
+	@$(OBJDUMP) $(OBJDUMP_FLAGS) $@ > target/sam4l/debug/storm.lst
+	@$(SIZE) $@
 
 target/sam4l/release/storm.sdb: target/sam4l/release/storm
 	@tput bold ; echo "Packing SDB..." ; tput sgr0
 	@$(SLOAD) pack -m "$(SDB_MAINTAINER)" -v "$(SDB_VERSION)" -n "$(SDB_NAME)" -d $(SDB_DESCRIPTION) -o $@ $<
 
+target/sam4l/release/storm.lst: target/sam4l/release/storm
+	@$(OBJDUMP) $(OBJDUMP_FLAGS) $^ > $@
+
 target/sam4l/release/storm.hex: target/sam4l/release/storm
 	@$(OBJCOPY) -Oihex $^ $@
 
+target/sam4l/debug/storm.hex: target/sam4l/debug/storm
+	@$(OBJCOPY) -Oihex $^ $@
+
 .PHONY: clean
 clean:
 	@cargo clean
 
+.PHONY: debug
+debug: target/sam4l/debug/storm
```

Comment:
But I don't mind changing those numbers... good to leave enough room for an app, but that's fungible anyway